examples/tts/conf/hifigan/hifigan.yaml

```diff
@@ -36,23 +36,31 @@ model:
     lr: 0.0002
     betas: [0.8, 0.99]
 
+  sched:
+    name: CosineAnnealing
+    min_lr: 1e-5
+    warmup_ratio: 0.02
+    max_steps: ???
+
   denoise_strength: 0.0025
-  max_steps: 2500000
 
 trainer:
   gpus: -1 # number of gpus
-  max_steps: ${model.max_steps}
+  max_epochs: ???
   num_nodes: 1
   accelerator: ddp
   accumulate_grad_batches: 1
   checkpoint_callback: False  # Provided by exp_manager
   logger: False  # Provided by exp_manager
+  flush_logs_every_n_steps: 200
   log_every_n_steps: 100
-  check_val_every_n_epoch: 10
-  automatic_optimization: false
+  check_val_every_n_epoch: 1
```

Comment:
This seems far too frequent. Can we leave it at 10?

Reply:
+